src/Admin.php

```diff
@@ -25,4 +25,89 @@ public static function admin_notices() {
     <?php }
   }
 
+  /**
```

Comment:
We are trying to structure our newer plugins into feature classes.  The backlinks are a new feature.  Can you move the new code into a new AdminBacklinks class?